ispyb/model/__future__.py

```diff
@@ -479,6 +484,42 @@ def _get_detector(self):
         self._data = cursor.fetchone()
 
 
+def _get_sample(self):
+    # https://jira.diamond.ac.uk/browse/SCI-XXXX
+    with _db_cc() as cursor:
+        cursor.run(
+            "SELECT blSampleId, name, "
+            "crystalId, containerId "
+            "FROM BLSample "
+            "WHERE blSampleId = %s",
+            self.id,
+        )
+        self._data = cursor.fetchone()
+
+        # Get the dcids associated with this sample group
+        cursor.run(
+            "SELECT dataCollectionId "
+            "FROM DataCollection "
+            "WHERE BLSAMPLEID = %s " % self._data["blSampleId"]
```

Comment:
```suggestion
            "WHERE BLSAMPLEID = %s ", self._data["blSampleId"]
```

should this be a `,`?